lib/ansible/modules/windows/win_copy.py

```diff
@@ -44,59 +33,89 @@
     - Remote absolute path where the file should be copied to. If src is a
       directory, this must be a directory too.
```

Comment:
There is only 1 case where it doesn't create directories, if dest is the path to a folder ending with `\` or `/`, I've since updated the PR to fail on this scenario as well as add a test for it.